budget/README.md

```diff
@@ -0,0 +1,37 @@
+# budget
+
+A Wing library for working with [AWS Budgets](https://docs.aws.amazon.com/cost-management/latest/userguide/budgets-managing-costs.html)
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+`sh
+npm i @winglibs/budget
+`
+
+## Usage
+
+**⚠️ The budget refers to the entire account and not just for the current project!**
+
+Add your budget to the code:
+
+```
+bring budget;
+
+let fifyCents = new budget.Budget(
```

Comment:
```suggestion
let fifyCents = new budget.Alert(
```

Maybe call this "alert"?

Reply:
I don't mind, but if I want to add support for action in the future, something like this:

```
let b = new budget.Budget(
    ...
);

b.addAction(inflight () => {
    ...
})
```

The name `Alert` doesn't seem right.

Reply:
I am not sure it doesn't make sense. `alert.addAction()` sounds okay to me.

Your call.